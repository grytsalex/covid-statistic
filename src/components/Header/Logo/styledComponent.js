import styled from 'styled-components';

export const LogoImage = styled.img`
  width: ${({ width }) => width || 'auto'};
  height: ${({ height }) => height || 'auto'};
`;

export const LogoTitle = styled.h1`
  font-size: ${({ fontSize }) => fontSize || '72px'};
  line-height: 84px;
  text-transform: uppercase;
  font-weight: bold;
`;

export const LogoWrapper = styled.div`
  width: 100%;
  display: flex;
  align-items: center;
  margin: 40px 0 0 60px; 
`;