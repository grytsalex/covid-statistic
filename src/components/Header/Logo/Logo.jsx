import React from "react";
import { LogoImage, LogoWrapper, LogoTitle } from "./styledComponent";
import img from "../../../assets/logo.jpg";

export const Logo = () => {
  return (
    <LogoWrapper className={"logo-wrapper"}>
      <LogoImage src={img} />
      <LogoTitle className={"logo-title"}>statistic</LogoTitle>
    </LogoWrapper>
  );
};
