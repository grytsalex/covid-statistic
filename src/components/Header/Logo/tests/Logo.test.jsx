import React from "react";
import { create } from "react-test-renderer";
import { Logo } from "../Logo";

describe("Logo", () => {
  it("should match snapshot", () => {
    const component = create(<Logo />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
