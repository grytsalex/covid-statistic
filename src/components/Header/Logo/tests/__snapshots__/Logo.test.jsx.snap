// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match snapshot 1`] = `
<div
  className="sc-dlnjwi dNVBYq"
>
  <img
    className="sc-bdnxRM cKwECw"
    src="logo.jpg"
  />
  <h1
    className="sc-gtsrHT bICckE"
  >
    statistic
  </h1>
</div>
`;
